menu/drivers/ozone/ozone.h

```diff
@@ -73,6 +73,8 @@ typedef struct ozone_handle ozone_handle_t;
 #define OZONE_TICKER_SPACER "\xE2\x80\x83\xE2\x80\xA2\xE2\x80\x83"
 #endif
 
+bool missing_thumbnail;
```

Comment:
no need for this to be global to the whole program

Reply:
actually it will have to be moved to be "even more global" if we want to reuse the code in other drivers.
so I guess I should go the fake C++ route, adding a variable to retroarch.c and "getters and setters"